module.yaml

```diff
@@ -22,7 +21,7 @@ versions:
   dependencies:
     - id: resource_group
       refs:
-        - source: github.com/terraform-ibm-modules/terraform-ibm-toolkit-resource-group
+        - source: github.com/cloud-native-toolkit/terraform-ibm-resource-group
```

Comment:
why was this changed to the old git repo url?

Reply:
Looks like this was picked up from an older commit so unlikely that it is done on purpose. We can review with @daniel-butler-irl when he's back.